### PR TITLE
Asyncronous api calls

### LINKED ISSUE #30: Asynchronous API Calls
Build functionality for facade to make asynchronous vs. synchronous API calls to reduce response time.

- [ ] Update to README (FE screenshots, BE API endpoint documented)

### PR DESCRIPTION
**Pull request by the-color-bliu**
- [x] Updated Files: 
* trip_facade
* development.rb

- [ ] Routes Added: N/A

- [x] Repo Representation: Back End

- [x] All Tests are passing

- [ ] Updated README.md with added information

- [x] Collaborators: Aidan Murray + Brian Liu

closes #30

### CODE REVIEW

File: app/facades/trip_facade.rb [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+        end
       end
-      trips #return value: an array of Trip Objects
+      threads.map(&:value).flatten
+      threads.each do |t|
+        @trips << t[:trip]
+        t.exit
+      end
+
+      @trips # return value: an array of Trip Objects
```
Comment (JesseMellinger) [inline app/facades/trip_facade.rb:26]: Okay, that's pretty cool.

**Review by JesseMellinger — APPROVED** [review]
This looks great! Nice work.

**Review by cpfergus1 — APPROVED** [review]
Looks good but we should DRY up this code in refactor later
Thanks guys!

**Review by Todd-Estes — APPROVED** [review]

**Review by stillsheryl — APPROVED** [review]
This looks amazing! I see that you didn't add to the README, but I think this would be a great thing to explain a bit further or add some info (screenshot?) about!

**Pull request merged**